### ISSUE TITLE
loss modification

### ISSUE THREAD

**Issue by zhangxixi0904** [layumi/Person-reID-triplet-loss#4]
Hi！I find there is a loss item name 'reg'. What dose it mean?  

reg = torch.sum((1+nscore)**2) + torch.sum((-1+pscore)**2)
loss = torch.sum(torch.nn.functional.relu(nscore + opt.margin - pscore))  #Here I use sum
loss_triplet = loss + opt.alpha*reg

**Comment by layumi** [comment]
It is a trick. 
But I found it does not work well, so I does not use it (opt.alpha=0).

**Comment by zhangxixi0904** [comment]
thanks for your reply
